packages/core/legos/package.json

```diff
@@ -127,6 +129,7 @@
     "@chainlink/trueusd-adapter": "*",
     "@chainlink/twelvedata-adapter": "*",
     "@chainlink/unibit-adapter": "*",
+    "@chainlink/uniswap-v2-adapter": "*",
```

Comment:
Could you please rebase your PR? Looks like there's a few changes here that are already in `develop`.

Reply:
done